letsencrypt-auto-source/pieces/bootstrappers/suse_common.sh

```diff
@@ -1,6 +1,6 @@
 # If new packages are installed by BootstrapSuseCommon below, this version
 # number must be increased.
-BOOTSTRAP_SUSE_COMMON_VERSION=1
+BOOTSTRAP_SUSE_COMMON_VERSION=2
```

Comment:
We haven't documented this anywhere, but bumping this version number is what causes `certbot-auto` to refuse to upgrade itself until it is running interactively and can prompt the user for approval to install new packages.

In this case, I think it's overkill and we should leave this at 1 because if you've successfully created a venv in the past, you should be able to do so again without installing new Python packages.

Reply:
Ok, I did not got it the first time. So the increment is only relevant when in the exact same OS system, some packages need to be changed. I revert the increment.

Reply:
That's correct. The only time we've ever made use of this functionality is to switch CentOS 6 from Python 2.6 to Python 3.4. In that case, rather than bumping the version for all Red Hat OSes, we created a [new bootstrap script and version for CentOS 6](https://github.com/certbot/certbot/blob/master/letsencrypt-auto-source/pieces/bootstrappers/rpm_python3.sh) to limit the need to "rebootstrap" to those OSes.